syntax = "proto3";

// The greeting service definition.
service Department {
    // Sends a greeting
    rpc Search (DepartmentRequest) returns (DepartmentReply) {
    }
}

message DepartmentRequest {
    string search_term = 1;
}

message DepartmentReply {
    string department_name = 1;
    string department_id = 2;
    string user_id = 3;
}